src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml

```diff
@@ -0,0 +1,49 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/dataset.h5ad"
+info:
+  label: "Dataset"
+  summary: "The dataset to pass to a method."
+  slots:
+    layers: 
+      - type: integer
+        name: counts
+        description: Raw counts
+        required: true
+    obs:
+      - type: string
+        name: label
+        description: Cell type annotation.
+        required: true
+    uns:
+      - type: string
+        name: dataset_id
+        description: "A unique identifier for the dataset"
+        required: true
+      - name: dataset_name
+        type: string
+        description: Nicely formatted name.
+        required: true
+      - type: string
+        name: dataset_url
+        description: Link to the original source of the dataset.
+        required: false
+      - name: dataset_reference
+        type: string
+        description: Bibtex reference of the paper in which the dataset was published.
+        required: false
+      - name: dataset_summary
+        type: string
+        description: Short description of the dataset.
+        required: true
+      - name: dataset_description
+        type: string
+        description: Long description of the dataset.
+        required: true
+      - name: dataset_organism
+        type: string
+        description: The organism of the sample in the dataset.
+        required: false
+      - type: string
+        name: ccc_target
+        description: The cell type which is intended to receive the communicational signal. This vector needs to be of the same length as `ccc_source`.
+        required: true
```

Comment:
You require the `ccc_target` in `uns` of the dataset but it is not in the dataset loader:
https://github.com/vishnu-vasan/openproblems-v2/blob/eaac6f6fc6b6d619a268bd6b7305752fa4dfba24/src/datasets/api/file_raw.yaml#L175

Add it also here but do not set it as required as this is not required for all datasets and tasks.

Reply:
Got you, i will add it over there